docs/source/install/docker.rst

```diff
@@ -0,0 +1,40 @@
+Docker
+======
+
+`Docker <https://docs.docker.com/get-docker/>`__ uses containers to create virtual
+environments that isolate a ASReview LAB installation from the rest of the system.
+
+ASReview LAB Docker requirements
+--------------------------------
+
+`Install Docker <https://docs.docker.com/get-docker/>`__ on your local `host` machine.
+
+Download an ASReview LAB Docker image
+-------------------------------------
+
+The official ASReview LAB Docker images are located in the `asreview/asreview
+<https://hub.docker.com/r/asreview/asreview>`__ Docker Hub repository.
+
+Open `Terminal
+<https://support.apple.com/guide/terminal/open-or-quit-terminal-apd5265185d-f365-44cb-8b09-71a064a42125/mac>`__
+(macOS) or `Command Prompt
+<https://www.lifewire.com/how-to-open-command-prompt-2618089>`__ (Windows) and enter:
+
+.. code-block:: bash
+
+    docker pull asreview/asreview  # latest stable release
```

Comment:
No need to pull, isn't it? You can skip this. See https://github.com/asreview/asreview/tree/master/docker#quickstart-using-docker

Reply:
Indeed, if the image is not installed, `docker run` will pull anyway. My question is: maybe users should be aware of this download step? The downloaded image is quite a large file.

Reply:
Docker users will probably know. They can read about Docker when installing Docker or from the various resources online.